WCFDataService/Client/System/Data/Services/Client/Metadata/ClientPropertyAnnotation.cs

```diff
@@ -140,6 +144,19 @@ internal ClientPropertyAnnotation(IEdmProperty edmProperty, PropertyInfo propert
                 instance,
                 value).Compile();
 
+            if (backingField == null)
```

Comment:
I will change ClientEdmModel.FindBackingField() so it will only return a value for entity collection and complex properties. This would solve this as well, agreed?